README.md

```diff
@@ -9,23 +9,106 @@ support multiple currencies in the future.
 
 ## Features [↑](#-ariadne)
 
-* [x] Create a new HD wallet (BIP-32), account or address
-* [x] Delete or rename wallets/accounts
-* [x] Import secret keys used by Daedalus
+* [x] Wallet management
 * [x] BIP-39 mnemonics
-* [ ] BIP-44 compatibility
-* [x] Get the balance of a wallet/account/address
-* [ ] Get transaction history
-* [x] Send a transaction from a wallet
+* [x] BIP-44 compatibility
+* [ ] Compatibility with Daedalus
+* [x] Balances of wallets, accounts, address
+* [ ] Transaction history
+* [x] Transaction sending
 * [x] Spending passwords
 * [x] Optimized block storage (compared to Daedalus)
-* [x] All actions correspond to commands in the [Knit](knit/README.md)
-language and can be typed in a console
+* [x] [Knit](knit/README.md) language
 * [x] Notifications about new software versions
 * [ ] Automatic software updates
 * [ ] Integration with hardware wallets
 * [ ] Staking and delegation
 
+### Wallet management
+
+Create or delete an HD wallet (BIP-32) or an account inside a
```

Comment:
I would expand HD acronym here.
I, for one, forget it's meaning every time.

Reply:
@goodsoft care to approve? :)